src/services/administration/iam/policy/policy-create/PolicyCreatePage.vue

```diff
@@ -29,17 +40,26 @@
             </div>
             <div class="policy-create-contents">
                 <!--            song-lang-->
-                <p-field-group label="Content" />
-                <p-text-editor :code="code" />
+                <p-field-group label="Permissions" required />
+                <p-text-editor :code.sync="code" @update:code="handleCodeUpdate" />
```

Comment:
sync는 양방향 지원이 됩니다.
아래의 handleCodeUpdate 내용을 보니 state.code에 변경사항을 적용해주고 있더군요.
결국 같은 일을 두번하는 것입니다.

```suggestion
                <p-text-editor :code="code" @update:code="handleCodeUpdate" />
```

or

```suggestion
                <p-text-editor :code.sync="code" />
```

Reply:
validation 훅 사용은 RoleUpdateForm에 성모님이 하신거 보면 될듯요!